firebaseFunctions/src/index.ts

```diff
@@ -338,3 +338,28 @@ export const scheduledFirestoreBackup = functions.pubsub
         throw new Error("Creating backup failed");
       });
   });
+
+/**
+ * Function responsible for running automated data update fetch from DeepDAO API.
+ * It runs every 24 hours.
+ * It stores a map of organizations data inside a Google FireStore collection - `deep-dao`.
+ * The API-Url and API-Key are specified in `firebaseFunctions/.env` file
+ */
+export const scheduledDeepDaoOrganizationListUpdate = functions.pubsub
+  .schedule("every 24 hours")
+  .onRun(async () =>
+  {
+    return await deepDaoOrganizationListUpdate(firestoreAdminClient, functions);
+  });
+
+/**
+ * Manually invoke DeepDAO data update
+ */
+export const callDeepDaoAPI = functions.https.onRequest(
```

Comment:
Should we keep it after work on this is finished? If we will never intended to deploy this function, please delete it before merging the PR :)